README.md

```diff
@@ -8,3 +8,10 @@
 * [Blog](./blog): A simple blog example with one author, three blog entries and some images.
 * [The Example App](./the-example-app): An example space simulating an online learning platform. This space is used in [The node.js example app](https://github.com/contentful/the-example-app.nodejs).
 
+## Changing a model
+
+Any breaking changes to the model should be done in a way that prevents example apps that are rendering this model from breaking. Make breaking changes by duplicating the model directory and appending the appropriate version number as a suffix to the new directory. For instance, if you want to delete a field from an entry in the `Blog` content model, please duplicate the `blog` directory with the name blog-v2 and then make the appropriate changes.
+
+## Changing content (no model changes)
+
+If you simply want to change the values of some content fields, please go ahead and change them in the current version(s) of the relevant export.json file.
```

Comment:
Nice, but please mention in another section that a git tag must be created and a github release done since the CLI seed command checks for latest github releases, not latest commit 🙇